openesdh-repo/src/main/java/dk/openesdh/repo/webscripts/AbstractRESTWebscript.java

```diff
@@ -18,10 +17,7 @@
     private static final String STORE_ID = "store_id";
     private static final String STORE_TYPE = "store_type";
 
-    @Override
-    public void execute(WebScriptRequest req, WebScriptResponse res) throws
-            IOException {
-        Map<String, String> templateArgs = req.getServiceMatch().getTemplateVars();
+    protected NodeRef getNodeRef(WebScriptRequest req, Map<String, String> templateArgs) {
```

Comment:
I suppose this method could try to retrieve the nodeRef from params using both caseId or STORE_TYPE+STORE_ID+NODE_ID.
Otherwise there is ugly code duplication of the method retrieving nodeRef by caseId.